.github/workflows/release-repository.yaml

```
@@ -38,27 +38,27 @@ jobs:
       - name: Catkin release - create PR
         run: |
           gh pr create \
-          --base ${{ github.event.inputs.release_candidate }} \
+          --base ${{ inputs.release_candidate }} \
           --head ${{ steps.catkin_release.outputs.created_branch }} \
           --title "Release ${{ steps.catkin_release.outputs.version}}" \
           --body "This PR incorporates package(s) version and changelog update."
 
       - name: Catkin release - merge PR
-        if: ${{ fromJSON(github.event.inputs.automatic_mode) == true }}
+        if: ${{ fromJSON(inputs.automatic_mode) == true }}
```

actionlint (1.7.4)

1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)